libraries/engage/product/components/FeaturedMedia/helpers.js

```diff
@@ -0,0 +1,42 @@
+import { APP_PLATFORM } from '@shopgate/pwa-common/constants/Configuration';
+import { OS_ANDROID } from '@shopgate/pwa-common/constants/Device';
+import configuration from '@shopgate/pwa-common/collections/Configuration';
+
+const qualities = {
+  jpeg: 75,
+  png: 100,
+  webp: 70,
+};
+
+const defaultParams = {
+  width: 440,
+  height: 440,
+  quality: qualities.jpeg,
+  fill: 'fff',
+  format: 'jpeg',
+};
+
+/**
+ * Append platform dimensions and quality to image url.
+ * @param {string} url image url
+ * @param {Object} [params=undefined] params
+ * @returns {string}
+ */
+export const buildFeaturedImageUrl = (url, params) => {
```

Comment:
Please rename this helper function (and all its usages) to `buildMediaImageUrl` because it's used for all image media and not only for featured ones.